backend/controller/lmevaljob_controller.go

```diff
@@ -185,17 +201,27 @@ func (r *EvalJobReconciler) constructOptionsFromConfigMap(configmap *corev1.Conf
 	if v, found := configmap.Data[DriverServiceAccountKey]; found {
 		r.options.DriverServiceAccount = v
 	}
+	if v, found := configmap.Data[PodCheckingIntervalKey]; found {
+		if d, err := time.ParseDuration(v); err == nil {
+			r.options.PodCheckingInterval = d
+		}
+	}
+	if v, found := configmap.Data[ImagePullPolicyKey]; found {
+		if p, found := pullPolicyMap[v]; found {
```

Comment:
If I'm reading the docs right, I think we just need the containment check and then the key and value are actually the same string.

Also, should we error out if it's an invalid value?

Reply:
let me know if you want me to error out here.

Reply:
I think I'm fine with not erroring out, but I think we need a way of warning that there's an invalid value if we don't error out. Silent errors are a killer to debug (e.g. minor typos, case sensitivity, etc)

Reply:
sent error logs for this case now